client/App.jsx

```diff
@@ -22,13 +22,13 @@ const App = () => {
 
   const sortReviews = (sort, list) => {
     if ((!sort || sort === 'Newest') && !list.length) {
-      axios.get(`/${id}`)
+      axios.get(`/reviews/${id}`)
         .then((result) => {
           setReviews(result.data);
         })
         .catch((err) => console.log('error ', err));
     } else {
-      axios.get(`/sort/${id}/${sort || 0}/${JSON.stringify(list)}`)
+      axios.get(`/reviews/sort/${id}/${sort || 0}/${JSON.stringify(list)}`)
```

Comment:
This is an interesting use of a conditional inside of a template literal. I'd almost move it out for readability, but it's probably not necessary.

Reply:
It is a strange thing.  John did some strange stuff.